sdk/common/src/main/java/com/mesosphere/sdk/offer/CommonTaskUtils.java

```diff
@@ -448,8 +462,6 @@ public static TaskInfo packTaskInfo(TaskInfo taskInfo) {
 
             if (taskInfo.hasCommand()) {
                 executorInfoBuilder.setCommand(taskInfo.getCommand());
-            } else {
-                executorInfoBuilder.setCommand(CommandInfo.getDefaultInstance());
```

Comment:
(ensures that if the original TaskInfo lacked a command, the packed+unpacked version lacks it too)